source/common/quic/envoy_quic_client_stream.cc

```diff
@@ -228,6 +228,22 @@ void EnvoyQuicClientStream::OnStreamFrame(const quic::QuicStreamFrame& frame) {
   quic::QuicSpdyClientStream::OnStreamFrame(frame);
 }
 
+bool EnvoyQuicClientStream::OnStopSending(quic::QuicResetStreamError error) {
+  // Only called in IETF Quic to close read side.
+  ENVOY_STREAM_LOG(debug, "received STOP_SENDING with reset code={}", *this, error.internal_code());
```

Comment:
Plz add `stats_.rx_reset_.inc();`

Reply:
Added, thanks.

Reply:
BTW adding here seems to break multiplexed_upstream_integration_test so I moved it closer to actual callback. It seems we shouldn't increment in the normal case after local stream end.

Reply:
That makes sense, the stats increments when the stream receives both RESET and STOP_SENDING.